### PR TITLE
tp-qemu: disable multi queue test on RHEL.6 host.

### PR DESCRIPTION
**Pull request by CongSmile**
as RHEL.6 host does not support multi queue.

Signed-off-by: Cong Li <coli@redhat.com>

### CODE REVIEW

**Comment by CongSmile** [comment]
id: 1202716

File: qemu/tests/cfg/mq_change_qnum.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - mq_change_qnum:
     only Linux
-    queues = 4
```
Comment (ypu) [inline qemu/tests/cfg/mq_change_qnum.cfg]: Why remove this?

Comment (CongSmile) [inline qemu/tests/cfg/mq_change_qnum.cfg]: Sorry, it's a mistake.

Have updated it.

Thanks a lot.

**Comment by xutian** [comment]
LGTM,

Acked-by:  Xu Tian <xutian@redhat.com>

**Comment by ypu** [comment]
LGTM

Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by ypu** [comment]
Thanks @CongSmile and @xutian will merge it

**Pull request merged**